### PR TITLE
chore: update README.md

### PR DESCRIPTION
**Pull request by slbsh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,9 +35,13 @@ Boolean types (case insensitive) are parsed as such:
...
-deps: `glew`
+## Packages
+| Repository | Package |
+|------------|---------|
+| AUR        | [zooc-git](https://aur.archlinux.org/packages/zooc-git/) |
 
+## Building
 ```sh
+# deps: glibc, glew, mesa, libx11, libxrandr, libxext
 make zooc clean
```
Comment (nick-bors) [inline README.md]: maybe this should also have make clean DESDIR="..." install?

File: README.md [hunk 2]
```diff
@@ -35,9 +35,15 @@ Boolean types (case insensitive) are parsed as such:
...
+# and optionally
+DESDIR="..." make install
```
Comment (nick-bors) [inline README.md]: wrong way cause make variable

Comment (slbsh) [reply README.md]: womp womp

**Review by nick-bors — APPROVED** [review]
Nice section of official packages. Maybe we can get cory to package this for chimera?

Also sorry about forgetting about this

**Pull request merged**